### PR TITLE
Update Plot.php

### PR DESCRIPTION
**Pull request by NawafCraft**
Nice Code

### CODE REVIEW

File: src/thebigsmileXD/SkyBlock/Plot.php [hunk 1]
```diff
@@ -40,7 +40,7 @@ public function isHelper($username) {
      */
     public function addHelper($username) {
         if (!$this->isHelper($username)) {
-            $this->helpers[] = $username;
+            $this->helpers[$username] = $username;
```
Comment (thebigsmileXD) [inline src/thebigsmileXD/SkyBlock/Plot.php]: Nope

**Comment by NawafCraft** [comment]
Why !

**Comment by TheKillerTacos** [comment]
What does this fix? and @thebigsmileXD, can I modify this plugin? I'll updating it, ill Give Credit to you (and ImagicalCorp if needed?)

**Comment by NawafCraft** [comment]
Thats Not Fix Thats Is Better Code And Small

**Comment by thebigsmileXD** [comment]
Your array will look like:
array(player=>player,mendj=>mendj,steve=>steve)

**Comment by NawafCraft** [comment]
Okey i know that and i started To Make API https://github.com/DevGames/MiniGames-API

**Pull request closed without merging**